CruCentralCoast/Community/CommunityGroups/CommunityGroupsVC.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  CommunityGroupsVC.swift
+//  CruCentralCoast
+//
+//  Created by Landon Gerrits on 5/16/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+
+class CommunityGroupsVC: UITableViewController {
+    
+    var dataArray: Results<CommunityGroup>!
```

Comment:
interesting format of an Array that i've never seen before!

Reply:
This is the return type when fetching the CommunityGroups from the Realm database. It acts like an array, but is kept up-to-date with the data in Realm

Reply:
Oh nice looks good!